CMakeLists.txt

```diff
@@ -114,7 +114,7 @@ endif()
 
 pbuilder_install_files(${CMAKE_INSTALL_PREFIX}/data
         ${PROJECT_SOURCE_DIR}/data/MuonFlux-HyperK-ThetaPhi.dat
-	${PROJECT_SOURCE_DIR}/data/biasprofile.dat
+      	${PROJECT_SOURCE_DIR}/data/biasprofile.dat
         ${PROJECT_SOURCE_DIR}/data/theta_cos15deg_FWHM.txt
         ${PROJECT_SOURCE_DIR}/data/DSNBFluxes.json
       	${PROJECT_SOURCE_DIR}/data/LightInjectors.json
```

Comment:
you probably want to install your new `data/` files

Reply:
👍